docs/tutorials/kubernetes-cluster-in-gpc-with-gardener/_index.md

```diff
@@ -0,0 +1,71 @@
+---
```

Comment:
I don't see `images/copy_icon.png` being linked/used by this tutorial. Can we remove it or make the tutorial use it?